keepercommander/importer/imp_exp.py

```diff
@@ -384,11 +724,57 @@ def execute_import_folder_record(params, folders, records):
             rs_record.extend(import_rs.recordResponse)
 
     report_statuses('folder', (element.status for element in rs_folder))
-    report_statuses('record', (element.status for element in rs_record))
+    report_statuses('legacy record', (element.status for element in rs_record))
 
     return rs_folder, rs_record
 
 
+def record_status_to_str(status):  # type: (record_proto.RecordModifyStatus) -> str
```

Comment:
Again, not a big deal, but this could be done a bit faster with collections.default_dict.  Readability is probably a tossup.

Reply:
I did not know about defaultdict in python. Thanks